README.md

```diff
@@ -73,19 +73,24 @@ using a git repository and
 1.  Ensure that you either have **Owner** permissions for the whole Cloud
     project, or at minimum the **Monitoring Editor** role
 1.  Create a ts-bridge config with no metrics
-    *   `cd ~/gopath/src/github.com/google/ts-bridge/app; cp metrics.yaml.example metrics.yaml`
+    *   `cd ~/gopath/src/github.com/google/ts-bridge/app; cp
+        metrics.yaml.example metrics.yaml`
     *   Edit the yaml file, remove the datadog\_metrics sample content, and copy
         in the name of the project you just created into the
         stackdriver\_destinations section.
     *   Your `metrics.yaml` file should look like this:
-    ```
+
```

Comment:
The change in indenting and extra spacing breaks the list counter (i.e. it starts over at 1 after this ``` chunk)